core/src/main/java/edu/wpi/grip/core/operations/networktables/NTPublishOperation.java

```diff
@@ -28,9 +28,12 @@
     private final List<Method> ntValueMethods = new ArrayList<>();
 
     public NTPublishOperation(Class<T> type) {
-        this.table = NetworkTable.getTable("GRIP");
         this.type = checkNotNull(type, "Type was null");
 
+        // NetworkTables automatically initializes itself when we get a table.  We don't want this behavior.
+        table = NetworkTable.getTable("GRIP");
+        NetworkTable.shutdown();
```

Comment:
But I don't get an error saying that network tables isn't working now though.
The step should inform me that its invalid.

This is what it looks like when I have a network protocol of NONE set.
There should be a warning on the step.
![Not showing me a problem](https://dl.dropbox.com/s/9k7vo6m6q8yawgf/Screenshot%202016-01-08%2001.52.58.png)

This is an invalid fix.